ar-AR.json

```diff
@@ -399,6 +399,43 @@
 	"SETTINGS_HW_ACCELERATED": "Decoding acceleration",
 	"SETTINGS_HW_FULL": "Full acceleration",
 	"SETTINGS_HW_LEGACY": "Legacy",
+	"SUBSPICKER_AUDIO": "Audio",
+	"SUBSPICKER_SUBTITLES": "Addons Subtitles",
+	"SUBSPICKER_BLACK_BACKGROUND": "Use black background color",
+	"SUBSPICKER_VERTICAL_OFFSET": "Vertical Offset",
+	"SUBSPICKER_SIZE": "Size",
+	"SUBSPICKER_DELAY": "Delay",
+	"SUBSPICKER_EMBEDDED_SUBS": "Embedded Subtitles",
+	"MOBILE_PRIVACY_POLICY": "Privacy Policy",
+	"MOBILE_CRASH_REPORTING": "Crash Reporting",
+	"MOBILE_ZOOM_TO_FILL": "Zoom To Fill Screen",
+	"MOBILE_PASSTHROUGH": "Digital audio output (passthrough)",
+	"MOBILE_RUN_LOW_BATTERY": "Run on low battery",
+	"MOBILE_RUN_FOREGROUND": "Run as Foreground Service",
+	"MOBILE_PERFORMANCE": "PERFORMANCE & STABILITY",
+	"MOBILE_SERVER_ONLINE": "Online",
+	"MOBILE_SERVER_OFFLINE": "Offline",
+	"MOBILE_QR_EXPIRES": "QR Code expires in ",
+	"MOBILE_LOG_INTO": "Log in to your Stremio account ",
+	"MOBILE_SCAN_QR": "Scan QR code above or go to ",
+	"MOBILE_LINK_ACCOUNT": "Link Account",
+	"MOBILE_SPEEDPICKER_CANCEL": "Cancel",
+	"MOBILE_SPEEDPICKER_LABEL": "Select Playback Speed",
+	"MOBILE_EMPTY_LIBRARY": "Your library is empty",
+	"MOBILE_NO_ADDON_ACTIVE": "No Stremio addon active for this type.",
+	"MOBILE_ORGANIZER_WARNING": "The Stremio Organizer app provides limited functionality. For a list of playable streams and the full Stremio experience, use Stremio on desktop.",
+	"MOBILE_ADDON_LOADING": "is still loading",
+	"MOBILE_ADDONS_LOADING": "addons are still loading",
+	"MOBILE_ADDON": "Addon",
+	"MOBILE_CONFIGURE_ADDON": "Configure your addon",
+	"MOBILE_CONFIGURE_ADDON_LABEL": "Configure Addon",
+	"MOBILE_ORGANIZER_ADDONS_WARNING": "The Stremio Organizer app provides limited selection of addons. For the full Stremio experience, use Stremio on desktop.",
```

Comment:
this was correct as "provides a limited", only the `MOBILE_ORGANIZER_WARNING` should of been changed to "provides limited"